Feature empty reg form alert box

tnathu-ai opened the pull request:
+ Added alert box for submitting all correct required inputs

+ Added highlighted red box feedback for invalid input for required field

miketvo reviewed (changes requested):
- ~~I can't access your newly added modal box. Please put an instruction in the pull request on how to reproduce the desired effect.~~ `RESOLVED` ✅ 
- ~~The highlighting should apply to the whole .register-item div as I told you before. Please utilize my existing CSS classes.~~ `RESOLVED` ✅ 
- Reminder: Minimize the complexity you add to our codebase each commit. The more complex our project is, the more time and effort it will take to debug.
- **Skimming is not inherently bad**, you just **abuse** it too much. I just put this line here in case you didn't read the previous points. If so, **read them!**

tnathu-ai commented:
For clarification: Added alert box for submitting all correct required inputs means that alert box **only appears** when all the inputs satisfy the requirements and the user clicks on the submit button.

For the error feedback after submission, I've implemented visual feedback by highlighting the input in red
